spec/go_to_param_spec.rb

```diff
@@ -99,6 +99,18 @@ def self.helper_methods
       expect(controller.go_to_here_params(bar: 3)).to eq({ go_to: "/example?foo&bar=3" })
     end
 
+    it "makes sure the request path to go to is a valid utf8 string" do
+      weird_path = "\xE0\x80\x80"
+      weird_path.force_encoding("ASCII-8BIT")
+      weird_path << "weird\330stuff".force_encoding("ASCII-8BIT")
```

Comment:
Having everything in just one line make it harder to read :( 
It's not hugely important to test both cases (\330 ASCII-8BIT and the invalid UTF-8 chars), but it gave me confidence that both cases are handled in the same way - I'm not an expert with string encoding :(